torchaudio/transforms.py

```diff
@@ -705,6 +706,7 @@ class Fade(torch.nn.Module):
         fade_shape (str, optional): Shape of fade. Must be one of: "quarter_sine",
             "half_sine", "linear", "logarithmic", "exponential". (Default: ``"linear"``)
     """
+
```

Comment:
Please do not include changes unrelated to the PR.

Reply:
ok